### PR TITLE
Add vanish reasoning

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

Added reasoning for the vanish module

### CODE REVIEW

File: pajbot/modules/vanish.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class VanishModule(BaseModule):
             label="Command name (i.e. vanish)",
             type="text",
             required=True,
-            placeholder="Command name (no !)",
```
Comment (RAnders00) [inline pajbot/modules/vanish.py]: Any particular reason why you changed this? I assume in preparation for #697?

Comment (alazymeme) [reply pajbot/modules/vanish.py]: Yeah, but is also just seemed to sound better

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Added dank reasoning for the vanish module
```
Comment (RAnders00) [inline CHANGELOG.md]: I don't want to kill the fun but I would appreciate if this was worded in a more "technical" style if you understand me

File: pajbot/modules/vanish.py [hunk 3]
```diff
@@ -50,11 +50,19 @@ class VanishModule(BaseModule):
             default=0,
             constraints={"min_value": 0, "max_value": 5000},
         ),
+        ModuleSetting(
+            key="timeout_reason",
+            label="Timeout Reason",
+            type="text",
+            required=False,
+            placeholder="Point cost",
```
Comment (pajlada) [inline pajbot/modules/vanish.py]: bad placeholder

**Pull request merged**